app/controllers/events_controller.rb

```diff
@@ -43,7 +43,7 @@ def create
   def destroy
     event_id = params[:id]
     GoogleEvent.delete(session[:credentials], event_id)
-    redirect_to root_path
+    render json: {message: 'success'}
```

Comment:
No error handling?
What if the user isn't the owner of the event?
Frontend validation is not enough